client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table-expand/site-table-popover.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Popover } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { forwardRef } from 'react';
+import { useJetpackAgencyDashboardRecordTrackEvent } from '../../hooks';
+
+import './style.scss';
+
+const SiteTablePopover = (
+	{ savePreferenceType }: { savePreferenceType: () => void },
+	ref: any
```

Comment:
What do we need to make in order to specify the right type here? Do we need to use [forwardRef](https://react.dev/reference/react/forwardRef)?

Reply:
Leaving it as-is for now since there are a lot of unnecessary changes needed.